helm/ako/templates/configmap.yaml

```diff
@@ -25,6 +25,7 @@ data:
   subnetIP: {{ .Values.NetworkSettings.subnetIP | quote }}
   enableRHI: {{ .Values.NetworkSettings.enableRHI | quote }}
   subnetPrefix: {{ .Values.NetworkSettings.subnetPrefix | quote }}
+  nsxt1LR: {{ .Values.NetworkSettings.nsxt1LR | quote }}
```

Comment:
values.yaml defines it as `nsxT1LR` with an uppercase `T`. This won't match correctly. a we use it consistently as either `nsxt1LR` or `nsxT1LR`. 

Also should it be `nsxtT1LR`, one `t` for the nsx variant and the other for `Tier1` (like the env variable in statefulset to be NSXT_T1_LR).

Reply:
I think we can call it nsxtT1LR consistently. It's easier for people to type. Thanks for the catch, I will make the change.